docs/book/src/component-config-tutorial/testdata/project/PROJECT

```diff
@@ -4,6 +4,7 @@
 # More info: https://book.kubebuilder.io/reference/project-config.html
 componentConfig: true
 domain: tutorial.kubebuilder.io
+kubebuilderVersion: unknown
 layout:
 - go.kubebuilder.io/v4
```

Comment:
I think if we want to track this one we will need to have the CLI version used per plugin and scaffold.
See that we can init a project within the version X and then afterwords scaffold an API with N+2

Reply:
Ah, got it.
So it would be applied to go/v2 and go/v3 also?

Reply:
It should ONLY be applied to the Project Config v3.